lib/src/executor/runtime_host/tests.rs

```diff
@@ -179,6 +179,9 @@ fn execute_blocks() {
 
                     execution = req.inject_key(next_key.map(|nk| nk.into_iter()));
                 }
+                RuntimeHostVm::Offchain(_req) => {
+                    unimplemented!("Offchain calls are not supported yet.")
```

Comment:
```suggestion
                    unimplemented!()
```

They will intentionally never be supported in this test.